src/AppBundle/UseCase/UnfollowUser.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace AppBundle\UseCase;
+
+use Core\Repository\UserRepositoryInterface;
+
+final class UnfollowUser
+{
+    private $users;
+
+    public function __construct(UserRepositoryInterface $users)
+    {
+        $this->users = $users;
```

Comment:
If we stick to the naming with "Interface" as a suffix and to the concept that repository != collection, then I think "$users" variable should be called "$usersRepository" instead.

Reply:
:+1: